src/templates/app/config/database.yml.ecr

```diff
@@ -0,0 +1,6 @@
+<%= @database %>:
+<% if @database == "mysql" -%>
```

Comment:
Is this conditional needed?  Your setting the environment variable DATABASE_URL in the docker-compose.yml.